electrum/wallet.py

```diff
@@ -263,6 +263,28 @@ def save_db(self):
         if self.storage:
             self.db.write(self.storage)
 
+    def save_backup(self, pin_code=None):
+        new_db = WalletDB(self.db.dump(), manual_upgrades=False)
+        new_db.put('is_backup', True)
+        new_path = os.path.join(get_backup_dir(self.config), self.basename() + '.backup')
+        if new_path is None:
+            return
+        new_storage = WalletStorage(new_path)
+        if pin_code:
+            backup_pubkey = self.config.get('backup_pubkey')
+            if backup_pubkey is None:
+                return
+            w2 = Wallet(new_db, None, config=self.config)
+            w2.update_password(pin_code, None)
+            new_storage._encryption_version = StorageEncryptionVersion.USER_PASSWORD
+            new_storage.pubkey = backup_pubkey
```

Comment:
As discussed out of band, the wallets created this way will have storage-encryption but not keystore-encryption. `update_password` for these will not work